frontend/components/markup-modal/markup-modal-content.js

```diff
@@ -0,0 +1,204 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import clipboardCopy from 'clipboard-copy'
+import { staticBadgeUrl } from '../../lib/badge-url'
+import { generateMarkup } from '../../lib/generate-image-markup'
+import { Snippet2 } from '../snippet'
+import { H3, Badge } from '../common'
+import PathBuilder from './path-builder'
+import QueryStringBuilder from './query-string-builder'
+import RequestMarkupButtom from './request-markup-button'
+import CopiedContentIndicator from './copied-content-indicator'
```

Comment:
This duplicates some code with the render method in MarkupModal, however that code will all go away once we finish migrating the `exampleUrl`s. I don't think it's worth extracting the duplicated code in the meantime.